### PR TITLE
Create Hello

### PR DESCRIPTION
**Pull request by stake2**

### CODE REVIEW

**Comment by stake2** [comment]
@Lytasia 
Hello ^^

**Comment by stake2** [comment]
Hello, thank you for working on MonikAI to make it better.
Why the issues are deactivated?
How can I use the Firefox Extension/Addon if it does not exist on the https://addons.mozilla.org/ website?
I can not talk with you on your Github profile page, why?

@Lytasia

**Comment by Lytasia** [comment]
Heh, i really don't know why issue are desactivated, and why you can talk to me on my page, i don't use GitHub often, but i'll look what's wrong

And for the Firefox extension, i need to reupload it, i'll do it really soon

**Comment by Lytasia** [comment]
@stake2 
Good news, issue are reopened and MonikAI extension is on Firefox addon site

**Comment by stake2** [comment]
@Lytasia 
Thanks ^^
Why did Firefox refuse?
Can you upload the extension to the repo and link it in the README.md file?

**Comment by Lytasia** [comment]
"Collecting ancillary personal information (e.g., any data not explicitly required for the add-on’s basic functionality) is prohibited.
monikai.js line 5"

That's what they said

I'll upload the zip into the github repo and write how to install it today

**Comment by stake2** [comment]
@Lytasia 
Thank you so much! hehe
I'm waiting to the zip file

**Comment by stake2** [comment]
@Lytasia 
Hey
I managed to make a .xpi file for the extension, it is here:
https://github.com/stake2/monikai-firefox-extension/blob/master/MonikAI%20Extension.xpi

Firefox says that it can't install unverified addons, so you can turn the check off by reading this page:
https://www.techplusme.com/how-to-install-unverified-add-ons-in-firefox/

Or you can install the extension using the .zip file by reading here:
https://developer.mozilla.org/en-US/docs/Mozilla/Add-ons/WebExtensions/Your_first_WebExtension

tl;dr:
Go to about:debugging on Firefox:
https://developer.mozilla.org/en-US/docs/Tools/about:debugging

And click "Load Temporary Add-on", then select the zip file that you downloaded.

**Comment by Lytasia** [comment]
@stake2 thanks you !
I'm not good with everything related to internet, so you're saving me
I'll Maybe make an update that download and show the user how to install it

Wait until the End of the week and the update will be here

**Comment by stake2** [comment]
@Lytasia 
No problem hehe
I wouldn't develop MonikAI because I don't know C, C++ or C# (I don't know what MonikAI uses)
But I can help you with internet things, that's what I know, so there are these two forms of installing the addon
The one by disabling the Firefox extension check then installing the .xpi file

And the other one by temporarily installing the addon from the .zip file using the debugging page.
If you need any help with internet stuff just ask!
i'll be glad to help you ^^

**Comment by Lytasia** [comment]
It use C#, but it doesn't matter, not everyone know the same language

And it's less annoying to install it once and then, done, so i'll use the xpi method

And i'll ask you if I need help again

**Comment by stake2** [comment]
Okay, thanks for working on MonikAI ^^
